Sources/Tracking/autogenerated/AutoDependencyInjection.generated.swift

```diff
@@ -225,7 +216,58 @@ public class DITracking {
     }
 
     private var newQueue: Queue {
-        CioQueue(siteId: siteId, storage: queueStorage)
+        CioQueue(siteId: siteId, storage: queueStorage, runRequest: queueRunRequest, jsonAdapter: jsonAdapter,
+                 logger: logger, sdkConfigStore: sdkConfigStore)
+    }
+
+    // QueueRequestManager (singleton)
+    public var queueRequestManager: QueueRequestManager {
+        if let overridenDep = overrides[.queueRequestManager] {
+            return overridenDep as! QueueRequestManager
+        }
+        return sharedQueueRequestManager
+    }
+
+    private let _queueRequestManager_queue = DispatchQueue(label: "DI_get_queueRequestManager_queue")
+    private var _queueRequestManager_shared: QueueRequestManager?
+    public var sharedQueueRequestManager: QueueRequestManager {
+        _queueRequestManager_queue.sync {
+            if let overridenDep = self.overrides[.queueRequestManager] {
+                return overridenDep as! QueueRequestManager
+            }
+            let res = _queueRequestManager_shared ?? _get_queueRequestManager()
+            _queueRequestManager_shared = res
+            return res
+        }
+    }
+
+    private func _get_queueRequestManager() -> QueueRequestManager {
+        CioQueueRequestManager()
+    }
+
+    // QueueRunRequest
+    public var queueRunRequest: QueueRunRequest {
+        if let overridenDep = overrides[.queueRunRequest] {
+            return overridenDep as! QueueRunRequest
+        }
+        return newQueueRunRequest
+    }
+
+    private var newQueueRunRequest: QueueRunRequest {
+        CioQueueRunRequest(runner: queueRunner, storage: queueStorage, requestManger: queueRequestManager,
+                           logger: logger)
+    }
+
+    // QueueRunner
+    public var queueRunner: QueueRunner {
+        if let overridenDep = overrides[.queueRunner] {
+            return overridenDep as! QueueRunner
```

Comment:
Safe to use if let/guard let/ optional for downcasting or upcasting

Reply:
That file is automatically generated for us and the code is only executed from test classes so the SDK shouldn't crash at runtime from this code.